news/2024/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "The Followpoint: D I O, the Mastermind Behind osu! Tournaments"
```

Comment:
i think this title could be shorter

```suggestion
title: "The Followpoint: D I O, the osu! Tournament Mastermind"
```

from https://osu.ppy.sh/home/news/2023-11-19-community-contributors-2022-2023
```suggestion
title: "The Followpoint: D I O, the Charismatic Caster"
```